dlpy/applications.py

```diff
@@ -4001,3 +4028,147 @@ def InceptionV3(conn, model_table='InceptionV3',
             model = Model.from_table(conn.CASTable(model_table))
 
             return model
+
+
+def Faster_RCNN(conn, model_table='Faster_RCNN', n_channels=3, width=1000, height=496, scale=1,
+                norm_stds=None, offsets=(102.9801, 115.9465, 122.7717), random_mutation = 'none',
+                n_classes=20, anchor_num_to_sample = 256, anchor_ratio = [0.5, 1, 2], anchor_scale = [8, 16, 32],
+                base_anchor_size = 16, coord_type = 'coco', max_label_per_image = 200, proposed_roi_num_train = 2000,
+                proposed_roi_num_score = 300, roi_train_sample_num = 128, roi_pooling_height = 7, roi_pooling_width = 7,
+                nms_iou_threshold = 0.3, detection_threshold = 0.5, max_object_num = 50):
+    '''
+    Generates a deep learning model with the faster RCNN architecture.
+
+    Parameters
+    ----------
+    conn : CAS
+        Specifies the connection of the CAS connection.
+    model_table : string, optional
+        Specifies the name of CAS table to store the model.
+    n_channels : int, optional
+        Specifies the number of the channels (i.e., depth) of the input layer.
+        Default: 3
+    width : int, optional
```

Comment:
really, we have a skewed image :)

Reply:
Xindian gave me a VOC dataset that is in the size.